### DISCUSSION TITLE
GTV

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#664] [Q&A]
**Channel Name:** GTV
**Channel ID:** GTV.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air channel in the Philippines.

**Comment by hazsmine** [comment]
Hi can i have the code for this specifiv channel? thank you

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19663
